5_recognition.ipynb

```diff
@@ -310,18 +548,40 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "## レゴブロックの位置の取得"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "## Get the position of the Lego block"
```

Comment:
Ah... the "Lego block" vs. "Lego blocks" question. XD
Is there a single block or more than one.

Reply:
This part is "the Lego block" (or "a Lego block"). We detect the position of single Lego block".

Reply:
Hmm... 
So, if this section is a continuation from a previous section where a Lego block was already introduced, then "the Lego block" will be fine.

If, on the other hand, this section is a new section where the previous section is does not introduce a Lego block, then "a Lego block" would flow nicer.
:)